README.md

```diff
@@ -188,13 +203,23 @@ oc get pods -n compute-config
 **NOTE: Restart the `compute-operator-manager` pod
 if you make any changes to the ClusterRegistrar or HubConfig.  This will allow the operator to onboard the new hub config.**
 
+# Using
 ## Import a user cluster into controller cluster
-1. Verify you are logged into the controller cluster
+1. Create and enter a new workspace in kcp
+
+2. Create an APIBinding to the compute-apis APIExport
+
+Edit the file hack/compute/apibinding.yaml spec.reference.workspace.path to point at the workspace you created above in [Generating a kubeconfig for your kcp cluster](#generating-a-kubeconfig-for-your-kcp-cluster)
```

Comment:
In another pr @chrisahl added the hack files in gitignore, we have to remove them from the gitignore

Reply:
@itdove I recently added the new hack files to .copyrightignore